external/kafka-0-10/src/test/scala/org/apache/spark/streaming/kafka010/KafkaTestUtils.scala

```diff
@@ -20,25 +20,24 @@ package org.apache.spark.streaming.kafka010
 import java.io.{File, IOException}
 import java.lang.{Integer => JInt}
 import java.net.InetSocketAddress
-import java.util.{Map => JMap, Properties}
 import java.util.concurrent.TimeoutException
-
-import scala.annotation.tailrec
-import scala.collection.JavaConverters._
-import scala.util.control.NonFatal
+import java.util.{Properties, Map => JMap}
```

Comment:
I see, couldn't make that out from the Github display. The imports shouldn't move though.

I have a feeling it was put there so that some other module's tests would be able to see it more easily in non-test scope, but that's not great. You will probably want to figure out what if anything was referencing this. I know we've had problems like this before, so it could be a hack to work around an SBT/Maven behavior difference.